packages/w3up-client/package.json

```diff
@@ -143,7 +143,7 @@
     "mock:gateway-server": "PORT=5001 node test/helpers/gateway-server.js",
     "coverage": "c8 report -r html && open coverage/index.html",
     "rc": "npm version prerelease --preid rc",
-    "docs": "npm run build && typedoc --out docs-generated"
+    "docs": "pnpm build && typedoc --out docs-generated"
```

Comment:
Can we please not change these? If someone clones the repo and does not use pnpm we still want things to work.